src/ignitus-Shared/Components/ProfileCard/index.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable jsx-a11y/accessible-emoji */
+import React from 'react';
+import './Styles/index.scss';
+import DropDownButton from './DropDownButton';
+
+export const SocialIcons = () => (
+  <div className="card-links">
+    <i className="fa fa-linkedin-square" />
```

Comment:
I would suggest to use to follow this pattern to display icons, https://github.com/Ignitus/Ignitus-client/tree/develop/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Assets/ignitus-Icons
Use `font awesome` / `material icons` to download the svg.